README.md

```diff
@@ -28,3 +28,35 @@ about the current recursive call.
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. You do not need to prove that the invariant is correct. Add
 your answer to this markdown file.
+
+
+## Solution
+
+**Invariant**:
+Let fib(n) be the function that computes the nth Fibonacci number. The invariant can be stated as: For all $n > 1$, if fib(n-1) and fib(n-2) are correct, then fib(n) is also computed correctly.
```

Comment:
But how do you know that fib(n-1) and fib(n-2) are correct?

Reply:
because the first calculation outside of fib(0) and fib(1) are is f(2) which is calculated from the known correct values of fib(0) and fib(1) {0 and 1 } respectively. fib(n) is equal to fib(n-1) and fib(n-2) we can apply this directly. fib(2) = 0 + 1 = 1 which can be assumed to be correct as  fib(0) and fib(1) {0 and 1 } are defined as base cases. Subsequently we can then extend that logic as correctness is based on this invariant

Reply:
...so then you can assume that the previous two Fibonacci numbers are correct?

Reply:
oh I see what your asking, thank you I'll fix it and make that clarification